meson.build

```diff
@@ -23,6 +23,7 @@ docdir = join_paths(prefixdir, get_option('datadir'), 'doc', 'util-linux')
 mandir = join_paths(prefixdir, get_option('mandir'))
 runstatedir = '/run'
 execprefixdir = prefixdir
+sysconfdir = join_paths(prefixdir, get_option('sysconfdir'))
```

Comment:
You can use / instead of join_paths

Reply:
thanks! for now i'd probably keep join_paths as the code around this line also uses it.